pymongo/collection.py

```diff
@@ -2219,7 +2219,7 @@ def aggregate_raw_batches(self, pipeline, **kwargs):
         return self._aggregate(pipeline, RawBatchCommandCursor, 0,
                                None, False, **kwargs)
 
-    def watch(self, pipeline=None, full_document='default', resume_after=None,
+    def watch(self, pipeline=[], full_document='default', resume_after=None,
```

Comment:
I suggest changing all the `pipeline` defaults back to `None`. It's generally bad practice to use mutable default arguments so we avoid them (even though it doesn't really matter in this case since the first thing we do is take a deepcopy of the pipeline).

Also since `None` is no longer a valid value that means we technically broke compatibility.